modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -129,21 +129,8 @@ def representative
     end
     # V2 End
 
-    def veteran_first_name
-      header_field_as_string 'X-VA-First-Name'
-    end
-
-    def veteran_last_name
-      header_field_as_string 'X-VA-Last-Name'
-    end
-
-    def ssn
-      header_field_as_string 'X-VA-SSN'
-    end
-
-    def file_number
-      header_field_as_string 'X-VA-File-Number'
-    end
```

Comment:
Even though we have `veteran` and `claimant` instances of `Appellant` available in the NOD here, we were still directly accessing headers to get these pieces of data. I changed them so that they continue to behave the same but are delegated to those `Appellant` instances instead.